transcom/mymove
# MB-16448 update language in customer onboarding screens

## [MB-16448 Jira ticket](https://dp3.atlassian.net/browse/MB-16448) && [MB-16582 Jira ticket](https://dp3.atlassian.net/browse/MB-16582)

## Summary

This pull request updates the customer on-boarding screens to have clear, appropriate language. 

The main changes include:
- Removed these contractions: It's, you'll, can't, you're, don't, that's
- Renamed "things" to "personal property"
- Renamed "movers" to "Customer Care Representative"
- Made "Signature" bold on legal agreement
- Added additional context 

## Verification Steps for the Author and Reviewers

- Start the Milmove application 
- Create a new user
- During the customer onboarding screens, verify the language matches the designs and changes highlighted in this [file](https://docs.google.com/spreadsheets/d/1OJef6pq41YhNBMy7pDrQqqspcv_EXH-RfXJmMqNzfZw/edit#gid=0)
